Better ReadMe (#9)
upb-uc4/deployment: README.md

 - development
 - production

## Deploy UC4-Website

> - 	In Portainer restart necessary containers (mariaDb needs to be on IP **.4)

Reviewer: ```suggestion
- 	In Portainer: Restart necessary containers (mariaDb needs to be on IP **.4)
```

-	Start the internal apache2 server ``` /etc/init.d/apache2 start ```

## Deploy Vue Frontend

> - 	In Portainer update Ui-web image from vue frontend container

Reviewer: ```suggestion
- 	In Portainer: Update the ui-web image of the vue frontend container
```

export {serviceName in caps}_VERSION={version}
envsubst < services/{serviceName}.yaml | kubectl apply -f -
```

## NGINX

> The nginx configuration used on our server are shown in the [nginx folder](nginx).

Reviewer: ```suggestion
The nginx configurations used on our server are shown in the [nginx folder](nginx).
```
